### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by MesterPerfect** [hellzerg/optimizer#475]
Hello,
In the past, I raised an accessibility issue, and the developer responded very well. 
https://github.com/hellzerg/optimizer/issues/150

Now, I have another question 😊.

While using the Optimizer tool, for example, with the General tab, there are various services such as:

* Disable Office Telemetry
* Disable Mozilla Firefox Telemetry
* Disable HomeGroup
(and so on)

Usually, if a user wants information about any service, they can right-click with the mouse, and a window appears containing a multi-line edit with a description of the service.

However, this feature is not accessible with a screen reader.

I propose two solutions:

## First Solution:
While focusing on any of the mentioned items and pressing the applications key on the keyboard, the function to display the multi-line edit with a description of the service can be executed.

## Another Solution:
Passing text inside the multi-line edit to a screen reader via libraries like this one:

[Tolk Library](https://github.com/dkager/tolk)

or any other library the developer prefers to use)[]

Of course, the text scrolling function must be associated with a key on the keyboard and not activated automatically or randomly.

Thank you, @hellzerg, for this exceptional tool.

**Comment by gruianovac** [comment]
it should detect the current state .. like tcp ip optimizer 1 detect / current 2 optimal / suggested 3 default 
and idealy save current state or suggest it
